glue/dialogs/link_editor/qt/data_graph.py

```diff
@@ -162,8 +166,12 @@ def get_connections(dc_links):
     for link in dc_links:
         data1 = link.data1
         data2 = link.data2
+        if link.join_link:
+            join_link = True
+        else:
+            join_link = False
         if (data1, data2) not in links and (data2, data1) not in links:
-            links.append((data1, data2))
+            links.append((data1, data2, join_link))
```

Comment:
I wonder if instead of having join_link be a boolean we should have the third argument be a link_type which could be a string, then it could be currently either 'join' or some name we pick for the current kind of linking (I don't have strong opinions on what that should be currently, could be e.g. 'value' or something like that.

Reply:
That makes sense. For now I have 'join' and 'value' as the two link_types